server/server/src/Program.cs

```diff
@@ -1,37 +1,45 @@
 using server.src;
 using Microsoft.EntityFrameworkCore;
-using Microsoft.EntityFrameworkCore.SqlServer;
-
+using server.src.Task1;
+using Npgsql;
+using server.UserNamespace;
 namespace server
 {
     public class Program
     {
         public static void Main(string[] args)
         {
-            // **UNCOMMENT THIS ONCE YOU HAVE CREATED THE DATABASE**
-            //DatabaseManager databaseManager = new DatabaseManager("Server=localhost\\SQLEXPRESS01;Database=flash-read-db;Trusted_Connection=True;");
-            
             var builder = WebApplication.CreateBuilder(args);
-            var  MyAllowSpecificOrigins = "_myAllowSpecificOrigins";
 
-            builder.Services.AddCors(options =>
-            {
-                options.AddPolicy(name: MyAllowSpecificOrigins,
-                                policy  => {
-                                    policy.AllowAnyOrigin()
-                                            .AllowAnyMethod()
-                                            .AllowAnyHeader();
-                                    });
+            var MyAllowSpecificOrigins = "_myAllowSpecificOrigins";
+            builder.Services.AddCors(options => {
+                options.AddPolicy(
+                    name: MyAllowSpecificOrigins, 
+                    policy  => {policy.AllowAnyOrigin().AllowAnyMethod().AllowAnyHeader();}
+                );
             });
 
-
-            // Add services to the container.
-            // Learn more about configuring Swagger/OpenAPI at https://aka.ms/aspnetcore/swashbuckle
-            builder.Services.AddDbContext<FlashDbContext>(options => 
-                options.UseSqlServer("Server=localhost\\SQLEXPRESS01;Database=flash-read-db;Trusted_Connection=True;"));
             
+            try
+            {
+                var dataSourceBuilder = new NpgsqlDataSourceBuilder(ConnectionStringBuilder.BuildConnectionString());
```

Comment:
We should handle the case when we fail to connect to the database. The program can exit or run a loop trying to connect again.

Reply:
On it.